app/routes.py

```diff
@@ -1,2 +1,65 @@
-from flask import Blueprint
+from flask import Blueprint, jsonify, make_response, request
+from app import db
+from app.models.planet import Planet
 
+
+planets_bp = Blueprint("planets", __name__, url_prefix="/planets")
+
+
+@planets_bp.route("", methods=["GET", "POST"])
+def find_planets():
+    if request.method == "GET":
+        planets = Planet.query.all()
+        planets_response = []
+        for planet in planets:
+            planets_response.append({
+                "id": planet.id,
+                "name": planet.name,
+                "description": planet.description,
+                "color": planet.color
+            })
```

Comment:
We could reduce these lines of code using a helper method to turn the planet object into a dictionary in the Planets class:

```suggestion
    planets_response = []
    for planet in planets:
        planets_response.append(planet.to_dict())
```
And further refactor using a list comprehension :D 

```suggestion
        planets_response = [ planets_response.append(planet.to_dict()) for planet in planets ]

```